samples/host_verify/CMakeLists.txt

```diff
@@ -19,8 +20,14 @@ if (BUILD_HOSTVERIFY_INLINE)
   install(
     TARGETS host_verify
     EXPORT openenclave-targets
+    DESTINATION ${CMAKE_INSTALL_BINDIR})
+
+  install(
+    TARGETS host_verify
+    EXPORT openenclave-hostverify-targets
     DESTINATION ${CMAKE_INSTALL_BINDIR}
-    COMPONENT OEHOSTVERIFY)
+    COMPONENT OEHOSTVERIFY
+    EXCLUDE_FROM_ALL)
```

Comment:
@ryanhsu19 does `EXCLUDE_FROM_ALL` mean that host_verify is not built and installed in the host-verify package?

The purpose of https://github.com/openenclave/openenclave/pull/3014 was that this binary should be released with OpenEnclave, because it is useful to end users. If a separate host-verify package is created, with fewer dependencies (which is an excellent change!), then this tool should be included in this package as well.